flow/__init__.py

```diff
@@ -25,6 +25,22 @@
 from .util.misc import redirect_log
 from .operations import with_job
 
+# Enable flow environments on an opt-out basis. Users must set the relevant
+# configuration variable to avoid environment being imported. This may be done
+# on either a global or a local configuration level. The order of precedence is
+# local up to global.
+from signac import get_project as _get_project
+
+# Work up in the correct order
+try:
+    flag = _get_project().config['flow'].get('IMPORT_ENVIRONMENTS')
```

Comment:
There is no need to deal with any kind of logic concerning the origin of the configuration value here. Just use the `get_config_value()` function from the `util.config` module:
https://github.com/glotzerlab/signac-flow/blob/master/flow/util/config.py#L44

Reply:
Got it, thanks. I'll fix.